src/cli/console/exit.py

```diff
@@ -9,3 +9,8 @@ def exit_login_required():
 def exit_generic_error():
     error("Something went wrong!")
     exit(1)
+
+
+def exit_error_with_message(message: str):
```

Comment:
I am not sure if I would introduce this function. Instead I would use:
`console.error("ERROR MESSAGE" ,_exit=True)`